### ISSUE TITLE
[Q] Combinator with default values?

### ISSUE THREAD

**Issue by ocramz** [fimad/scalpel#62]
A Scraper might fail for multiple reasons; for example, a web page might not contain every field for certain query configurations.

What's a good approach for scraping optionally-present fields?

**Comment by fimad** [comment]
The way I prefer is by using `<|>`. For example: 

```haskell
scrapeStringLike "<a>1</a>" $ (text "b" <|> return "defaultValue")
```

We get `<|>` because `Scraper` is an instance of the [Alternative](https://hackage.haskell.org/package/base-4.10.1.0/docs/Control-Applicative.html#t:Alternative]) class.

**Comment by ocramz** [comment]
Indeed, I overlooked this. I like Alternative too but I was somewhat
expecting to see it in the examples.
Perhaps I'll isolate a snippet and send a PR.

Thanks
